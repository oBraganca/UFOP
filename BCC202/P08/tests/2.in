2
10
joana 32
pedro 27
mariana 42
lucas 19
carolina 36
rafael 25
laura 31
gustavo 29
isabela 23
fernando 40
9
sophia 27
liam 33
olivia 29
noah 35
emma 22
jackson 41
ava 26
aiden 31
isabella 24