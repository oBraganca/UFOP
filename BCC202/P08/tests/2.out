Teste 1:
carolina - 36
fernando - 40
gustavo - 29
isabela - 23
joana - 32
laura - 31
lucas - 19
mariana - 42
pedro - 27
rafael - 25

Teste 2:
aiden - 31
ava - 26
emma - 22
isabella - 24
jackson - 41
liam - 33
noah - 35
olivia - 29
sophia - 27